ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+      MasterUserPassword: !Sub '{{resolve:secretsmanager:${ghostDBRootUserSecret}:SecretString:password}}'
+      EnableHttpEndpoint: true
+      ScalingConfiguration:
+        AutoPause: true
+        MinCapacity: 1
+        MaxCapacity: 2
+      StorageEncrypted: true
+      VpcSecurityGroupIds:
+        - !Ref ghostDBSecurityGroup
+  ghostDBRootUserSecretAttachment:
+    Type: AWS::SecretsManager::SecretTargetAttachment
+    Properties:
+      SecretId: !Ref ghostDBRootUserSecret
+      TargetId: !Ref ghostDB
+      TargetType: AWS::RDS::DBCluster
+  ghostDBSubnetGroup:
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: !Sub
+        - Stack ${StackTagName} Environment ${EnvironmentTagName} Database ${ResourceName} VPC Subnets
+        - ResourceName: ghostDB
+      SubnetIds: !Ref DefaultVPCSubnets
+      Tags:
+        - Key: Name
+          Value: !Sub
+            - Stack ${StackTagName} Environment ${EnvironmentTagName} Database ${ResourceName} VPC Subnets
+            - ResourceName: ghostDB
+  ghostDBRootUserSecret:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Description: !Sub
+        - Root user login info for Stack ${StackTagName} Environment ${EnvironmentTagName} Database ${ResourceName}
+        - ResourceName: ghostDB
+      GenerateSecretString:
+        SecretStringTemplate: '{"username": "root"}'
+        GenerateStringKey: password
+        PasswordLength: 16
+        ExcludeCharacters: '"@/\'
+      Name: !Sub /${EnvironmentTagName}/${StackTagName}/ghostDB/RootUser
+  ghostDBSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Database Security Group
+      SecurityGroupIngress:
+        # CidrIp: 0.0.0.0/0
+        SourceSecurityGroupId: !Ref ghostAPISecurityGroup        
+        FromPort: 3306
+        IpProtocol: tcp
+        ToPort: 3306
+      Tags:
+        - Key: Name
+          Value: !Sub
+            - Stack ${StackTagName} Environment ${EnvironmentTagName} Database ${ResourceName} VPC Security Group
+            - ResourceName: ghostDB
+      VpcId: !Ref DefaultVPCId
+  ghostAPISecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: API Security Group
+      SecurityGroupIngress:
+        IpProtocol: tcp
+        SourceSecurityGroupId: !Ref ghostLBSecurityGroup        
+        FromPort: 2368
+        ToPort: 2368
+      Tags:
+        - Key: Name
+          Value: !Sub
+            - Stack ${StackTagName} Environment ${EnvironmentTagName} API ${ResourceName} VPC Security Group
+            - ResourceName: ghostAPI
+      VpcId: !Ref DefaultVPCId
+  ghostEFSSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: EFS Security Group
+      SecurityGroupIngress:
+        IpProtocol: tcp
+        CidrIp: 0.0.0.0/0
+        FromPort: 2049
+        ToPort: 2049
```

Comment:
I think this should be like the ghostDBSecurityGroup, in that it only allows ingress from the ghostAPISecurityGroup.

Reply:
Was this addressed? I can't tell...

Reply:
Nevermind, this was addressed!